Dockerfile

```diff
@@ -39,20 +39,13 @@ RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
 # Add go and oc binary directory to $PATH
 ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
 
-# Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
-ARG GIT_CHECKOUT_TARGET=$TNF_VERSION
-
```

Comment:
TNF_SRC_URL this argument can be specified to pass the code of any PR from the fork. 
you can as well pass the branch used in GIT_CHECKOUT_TARGET

I don't see the need for this change. Can you elaborate what's preventing you from passing the environment variables to your build system ?

Reply:
I got the same question. The only benefit I can see is to build local changes into an image without pushing to remote. Typically we run the binary directly for development/debugging local changes before pushing.